avocado/utils/memory.py

```diff
@@ -51,7 +51,7 @@ def _is_online():
                 return True
             return False
 
-    return wait.wait_for(_is_online, timeout=120, step=1) or False
+    return wait.wait_for(_is_online, timeout=10, step=1) or False
```

Comment:
would you please force-update the commit message? It wasn't 2s, it was up to 120s. The commit is fine, though, using 10s timeout seems more appropriate. Although you could perhaps even decrease the `step` to `0.2` or so to have the check every `0.2`s rather than once per 1s. It's up to you...